docs/general/recording-errors.md

```diff
@@ -76,50 +81,40 @@ messaging operation may involve sending multiple messages) and includes `error.t
-operation SHOULD have the same `error.type` value if the operation failed and SHOULD NOT
-include it if the operation succeeded.
-
-## Recording exceptions
-
-When an instrumented operation fails with an exception, instrumentation SHOULD record
-this exception as a [span event](/docs/exceptions/exceptions-spans.md) or a [log record](/docs/exceptions/exceptions-logs.md).
-
-It's RECOMMENDED to use the `Span.recordException` API or logging library API that takes exception instance
-instead of providing individual attributes. This enables the OpenTelemetry SDK to
-control what information is recorded based on application configuration.
-
-It's NOT RECOMMENDED to record the same exception more than once.
-It's NOT RECOMMENDED to record exceptions that are handled by the instrumented library.
-
-For example, in this code-snippet, `ResourceAlreadyExistsException` is handled and the corresponding
-native instrumentation should not record it. Exceptions which are propagated
-to the caller should be recorded (or logged) once.
-
-```java
-public boolean createIfNotExists(String resourceId) throws IOException {
-  Span span = startSpan();
-  try {
-    create(resourceId);
-    return true;
-  } catch (ResourceAlreadyExistsException e) {
-    // not recording exception and not setting span status to error - exception is handled
-    // but we can set attributes that capture additional details
-    span.setAttribute(AttributeKey.stringKey("acme.resource.create.status"), "already_exists");
-    return false;
-  } catch (IOException e) {
-    // recording exception here (assuming it was not recorded inside `create` method)
-    span.recordException(e);
-    // or
-    // logger.warn(e);
-
-    span.setAttribute(AttributeKey.stringKey("error.type"), e.getClass().getCanonicalName())
-    span.setStatus(StatusCode.ERROR, e.getMessage());
-    throw e;
-  }
-}
-```
+## Recording errors on logs
+
+When recording an error using logs:
+
+- MUST set [`EventName`][EventName] with a value that would normally be
+  used for an [`error.type`][ErrorType] attribute.
+- SHOULD set [`error.message`][ErrorMessage] attribute to add additional
+  information about the error, for example, an exception message.
+
+When an error is retried or handled and the overall operation completes successfully,
+it SHOULD still be recorded as an event record for diagnostic purposes.
+In such scenario, [`SeverityNumber`][SeverityNumber] MUST be below 17 (ERROR).
+
+When an error occurs outside the context of any span
+and it causes an operation to fail,
+the instrumentation SHOULD record it as an event record.
+In such scenario, [`SeverityNumber`][SeverityNumber] MUST be greater than
+or equal to 17 (ERROR).
+
+When an error occurs inside the context of a span
+and it causes an operation to fail,
+the instrumentation SHOULD NOT additionally record it as an event record.
+
+> [!NOTE]
+>
+> Applications that also want error event records corresponding to spans
```

Comment:
this note is not very clear.. consider rephrasing..
Did you mean to recommend users to use a SpanProcessor to convert SpanEvents to LogRecords?